pkgs/applications/networking/insync/default.nix

```
@@ -81,30 +82,43 @@ let
 
     # NB! This did the trick, otherwise it segfaults! However I don't understand why!
     dontStrip = true;
+  };
 
+  insync-fhsenv = buildFHSEnv { # ref: pkgs/build-support/build-fhsenv-bubblewrap/default.nix
```

```suggestion
  insync-fhsenv = buildFHSEnv {
```
that comment will eventually be outdated